demos/functions/views/demo/workbox-background-sync/sw.hbs

```diff
@@ -0,0 +1,16 @@
+{{{WORKBOX_SW_IMPORT}}}
+
+const bgSyncPlugin = new workbox.backgroundSync.Plugin('myQueueName');
+
+const networkOnlyStrategy = new workbox.strategies.NetworkOnly({
+  plugins: [bgSyncPlugin],
+});
+
+workbox.routing.registerRoute(new workbox.routing.Route(
```

Comment:
can this not be shorthand?

```javascript
workbox.routing.registerRoute(
  ({url}) => url.pathname === './example.txt',
  workbox.strategies.networkOnly({
    plugins: [bgSyncPlugin],
  })
);
```

Loses the extra variables and loses the `'Get'`

Reply:
done